Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM ocrd/core:edge
+FROM ocrd/core
 MAINTAINER OCR-D
 ENV DEBIAN_FRONTEND noninteractive
 
+ENV PREFIX=/usr/local
+
 WORKDIR /build
 COPY ocrd-im6convert .
 COPY ocrd-tool.json .
 COPY Makefile .
 
 RUN apt-get update && \
+    apt-get -y install apt-utils && \
     apt-get -y install --no-install-recommends \
     ca-certificates \
-    make \
-    imagemagick
+    make
+
+RUN make deps-ubuntu install
 
-RUN make install PREFIX=/usr/local
+ENV DEBIAN_FRONTEND teletype
```

Comment:
Is it intentional that you change `DEBIAN_FRONTEND` here (see also line 3)?

Reply:
Yes, it is. Exactly because we set it to non-interactive during installation, we must re-set it to interactive (for which `teletype` is the choice with minimal assumptions) afterwards.